src/Database/Query.php

```diff
@@ -266,6 +266,7 @@ public function getConnection(): Connection
      */
     public function execute(): StatementInterface
     {
+        $this->_statement = $this->_results = null;
```

Comment:
Why set `$this->_statement` to `null` when it's assigned a new value in the very next line?

Reply:
Because an error throws an exception and won't set it.